assets/scss/_styles_project.scss

```diff
@@ -1130,3 +1131,105 @@ ul.sectionlist > li:hover {
   padding: 0;
   font-size: 30px;
 }
+
+// Tutorials page start
+
+.search-panel__results.card-container {
+  max-width: 100%;
+}
+
+.ais-Hits-item {
+  display: flex;
+}
+
+#hits .ais-Hits-list {
+  list-style-type: none;
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.ais-Pagination-item {
```

Comment:
(opt) If you know how to toggle button size on square buttons, it seems like the buttons might be smaller than they appear. 

I don't know if this is the right object (here where I'm commenting) but I did notice at least in staging a lot of the buttons in the tutorials index/nav bar don't work for me if I try to click them, at first, they only have a small surface area. Tiny issue and non blocking, but if you could make the buttons _slightly bigger_ it would be completely unnoticeable. 

 Only the buttons that look like `<<` and `>>` are easily clickable, the rest I have to hover for a bit. Not sure how to fix, but might be a browser related issue on my end? 
 
 This bar is what I'm talking about:
<img width="979" alt="Screenshot 2023-08-31 at 10 32 44 AM" src="https://github.com/viamrobotics/docs/assets/20797913/ed6ae8cc-290b-42c4-80ac-00e6c891e92d">